### PR TITLE
feat: Sovled pre-commit errors

### PR DESCRIPTION
**Pull request by jet981217**
Made issue / pr template for python-tools
Implement installation script for module using this sub-module

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -14,10 +14,19 @@ echo "update python-tools"
 git pull origin main
 git checkout main
 
+
+echo "copy templates"
+cp -r .github/ISSUE_TEMPLATE ../.github
+cp -r .github/workflows ../.github
```
Comment (psycoder21) [inline update.sh:20]: Why this exists

Comment (psycoder21) [reply update.sh:20]: `cp -r .github/workflows ../.github`

**Review by psycoder21 — APPROVED** [review]
Great work!

**Review by yakhyo — COMMENTED** [review]
looks good, good job!

**Pull request merged**